modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -82,7 +85,10 @@ public SELF withPassword(String password) {
 
     public SELF withDatabaseName(String dbName) {
         throw new UnsupportedOperationException();
+    }
 
+    public SELF withUrlParam(String paramName, String paramValue) {
+        throw new UnsupportedOperationException();
```

Comment:
Why can't the implementation of this method be pulled up here?

Reply:
Yes, that's a good idea. I will fix it.

Reply:
This is done. I did it.